SPECS/tuned/tuned.spec

```diff
@@ -25,6 +25,7 @@ BuildRequires: python3-configobj
 BuildRequires: python3-dbus
 BuildRequires: python3-decorator
 BuildRequires: python3-pyudev
+BuildRequires: systemd-rpm-macros
```

Comment:
Just a note - requiring any systemd package has caused some circular dependency issues in the past ("`[grapher] Unfixable circular dependency found:`"). The buddybuild is not able to detect this. You might want to run a full package build to confirm that this change doesn't introduce a new dependency loop.

Reply:
Replacing this with `systemd-bootstrap-rpm-macros` to go along with what other packages in Mariner do

Reply:
@anphel31 - Can you take a look at it again. 

Successful run link - https://dev.azure.com/mariner-org/mariner/_build/results?buildId=505039&view=results